README.md

```diff
@@ -1,7 +1,32 @@
 # opentelemetry-dotnet-instrumentation
-.NET instrumentation and auto-instrumentation
+[![Gitter
+chat](https://badges.gitter.im/open-telemetry/opentelemetry-dotnet.svg)](https://gitter.im/open-telemetry/opentelemetry-dotnet-auto-instr)
 
-### Community Roles
+This project provides a .NET tracer that leverages the .NET profiling APIs to support .NET instrumentation and auto-instrumentation without requiring code changes to an application.
+
+## Status
+
+This project is in the early stages of development starting with an initial seeding of code from the [.NET Tracer for DataDog APM](https://github.com/DataDog/dd-trace-dotnet). Our current goal is to take the seeded tracer and update it to both listen to and generate OpenTelemetry tracing data. To accomplish this our current priorities are to:
```

Comment:
nit: The link text should read `Datadog APM` without the second capitalized D. It's unfortunate we can't fix that in our GitHub URL's 😄